CONTRIBUTING.md

```diff
@@ -181,6 +181,16 @@ However, it's not always that simple. Our [PR approval process](#pr-approval-pro
 
 One thing you will need to do only for your first pull request in Netdata is to accept the CLA. Until you do, the automated check for the CLA acceptance will be showing as failed. 
 
+#### PR title guidelines
+
+All verbs in PR titles must be in the past tense. E.g. "Added" instead of "Adds", "Fixed" instead of "Fixes".
```

Comment:
I _guess_ for some reason or another (_unsure where I got the habit from_) I've always put up PRs with verbs in the present tense because it reads better that way when opening/reviewing a PR. I'm a bit indifferent and okay with this but it'll take some mental retraining to accomplish. :)

Reply:
Agreed, but avoids having to modify EVERY SINGLE VERB when writing release notes.